src/app/campaign/store/actions/campaign-action.service.spec.ts

```diff
@@ -120,14 +120,12 @@ describe('CampaignActionService', () => {
         .milliseconds(0);
       const yesterday = moment.utc(today.valueOf()).subtract(1, 'days');
       service.updateFlightForm({ ...flightFixture, endAtFudged: yesterday, contractEndAtFudged: yesterday }, true, true);
-      expect(JSON.stringify(dispatchSpy.mock.calls[dispatchSpy.mock.calls.length - 1][0])).toEqual(
-        JSON.stringify(
-          campaignActions.CampaignFlightFormUpdate({
-            flight: { ...flightFixture, endAt: today, endAtFudged: yesterday, contractEndAtFudged: yesterday, contractEndAt: today },
-            changed: true,
-            valid: true
-          })
-        )
```

Comment:
Seems like a separate issue, but trying to address that tests are hanging on
```
(node:85206) UnhandledPromiseRejectionWarning: TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'element' -> object with constructor 'Object'
    |     property 'publicProviders' -> object with constructor 'Object'
    |     property 'ɵNgNoValidate_122' -> object with constructor 'Object'
    --- property 'parent' closes the circle
    at JSON.stringify (<anonymous>)
    at process.target._send (internal/child_process.js:778:25)
    at process.target.send (internal/child_process.js:676:19)
    at reportSuccess (/Users/sandikbarr/code/grove.prx.org/node_modules/jest-worker/build/workers/processChild.js:67:11)
(node:85206) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:85206) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
Gah, disregard. The tests hanging and this UnhandledPromiseRejectionWarning error were actually errors happening in the tests from velocity not being included in the spec's form group. When I ran those tests individually, the errors were reported, but this is what I saw when I ran the whole test suite.